app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorSchemes.kt

```diff
@@ -16,12 +16,11 @@ internal fun darkColorScheme() = androidx.compose.material3.darkColorScheme(
     onSurface = colorResource(R.color.text_primary), // used by ListItem -> headlineContent
     onSurfaceVariant = colorResource(R.color.text_secondary), // used by SearchBarDefaults.InputField placeholder, ListItem -> overlineContent
     inverseOnSurface = colorResource(R.color.session_item_text_on_highlight_background), // used by SessionCard
-    outline = colorResource(R.color.colorAccent), // used by SearchBarDefaults.InputField divider
```

Comment:
This "broke" the stroke color of the `OutlinedButton` on the search history screen:

<img width="270" height="200" alt="search-clear-before" src="https://github.com/user-attachments/assets/120648d8-9563-47e7-ba65-4d70e84a895b" /> <img width="270" height="200" alt="search-clear-after" src="https://github.com/user-attachments/assets/19f193d5-365f-462f-ac17-321d61fb243d" />

Reply:
Changing theme attributes in a way that's not really compatible with Material is fighting the system. "Outline" is supposed to not have a lot of color so not to draw the user's focus and to contrast well against the primary color.

I wouldn't change all outline buttons to be colorful. If you want to emphasize the button, you could use a filled or tonal button. If you just want to change this particular button, override the outline color for this button specifically.